openwisp_firmware_upgrader/hardware.py

```diff
@@ -39,6 +39,84 @@
                 'ar71xx-generic-ubnt-airrouter-squashfs-sysupgrade.bin',
                 {'label': 'Ubiquiti AirRouter', 'boards': ('Ubiquiti AirRouter',)},
             ),
+            (
+                'zbt-wg3526-16M-squashfs-sysupgrade.bin',
+                {'label': 'ZBT-WG3526 (16M)', 'boards': ('ZBT-WG3526 (16M)',)},
+            ),
+            (
+                'tl-wr2543-v1-squashfs-sysupgrade.bin',
```

Comment:
looks like `ar71xx-generic-` is missing

Reply:
ath79-generic-tplink_tl-wr2543-v1-squashfs-sysupgrade.bin

ath79 instead of ar7xx as Openwrt community is switching from ar7xx to ath79 all those devices.